app/src/main/java/com/growthfile/growthfileNew/MainActivity.java

```diff
@@ -1116,7 +1116,18 @@ public void onPageFinished(WebView view, String url) {
                     mWebView.evaluateJavascript("_native.setName('Android')", null);
                     hasPageFinished = true;
 
-                    FirebaseInstanceId.getInstance().getInstanceId()
+                    // aditya's code
+                    FirebaseInstanceId.getInstance().getInstanceId().addOnSuccessListener(new OnSuccessListener<InstanceIdResult>() {
+                        @Override
+                        public void onSuccess(InstanceIdResult instanceIdResult) {
+                            String token = instanceIdResult.getToken();
+                            Log.e("FCMToken", token);
+                            mWebView.evaluateJavascript("_native.setFCMToken('" + token + "')", null);
+                        }
+                    });
+                    // aditya's code ended

```

Comment:
This method is old . Why are you using it ?

https://firebase.google.com/docs/cloud-messaging/android/client